server/src/routes/auth.ts

```diff
@@ -144,16 +144,16 @@ auth.get("/verify-token", jwt, async (req, res, next) => {
 	}
 })
 
-auth.get("/signout", jwt, async (req, res, next) => {
+auth.post("/signout", jwt, async (req, res, next) => {
 	try {
 		const authHeader = req.headers["authorization"]
-		const token = authHeader && authHeader.split(" ")[1]
-		const blacklist = await new JWTDenyList({token:token})
-		await blacklist.save()
+		const token = authHeader && authHeader?.split(" ")[1]
```

Comment:
sorry for not being clear, I believe the use of `?` should remove the need for the `authHeader &&`.

`?` is shorthand for that, basically. It's conditional on the preceding object being truthy (i.e. `authHeader != false`)